app/services/sendinblue/lists.rb

```diff
@@ -0,0 +1,28 @@
+module Sendinblue
+  class Lists
```

Comment:
Nevidim nejaky dovod to balit takto do dalsich vrstiev, normalne by som toto volal priamo zo service.